chain/types/ethtypes/eth_types_test.go

```diff
@@ -52,6 +55,7 @@ func TestEthBigIntMarshalJSON(t *testing.T) {
 		{EthBigInt(big.NewInt(65)), []byte("\"0x41\"")},
 		{EthBigInt(big.NewInt(1024)), []byte("\"0x400\"")},
 		{EthBigInt(big.Int{}), []byte("\"0x0\"")},
+		{EthBigInt(big.MustFromString("4314726344238778862224669783019953265742676409691641697694925658285796229120")), []byte("\"0x98a0c6ef2224342b2a8727ae5111fd000000000000000000000000000000000\"")},
```

Comment:
Should marshal without the leading zeroes

Reply:
I can't construct an int that marshalls with leading zeros - are there any examples you can help me find?